Dockerfile.dapper

```
@@ -28,7 +34,11 @@ ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH_arm64=arm64 GOLAN
 RUN wget -O - https://storage.googleapis.com/golang/go1.16.4.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local
 
 RUN if [ "${ARCH}" == "amd64" ]; then \
-    curl -sL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s v1.27.0; \
+        curl -sL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s v1.27.0; \
+    fi
+# fugly workaround for https://bugzilla.suse.com/show_bug.cgi?id=1183043
+RUN if [ -x /usr/bin/zypper -a "${ARCH}" == "arm64" ]; then \
+        rpm -Uvh http://download.opensuse.org/distribution/leap/15.3/repo/oss/aarch64/binutils-gold-2.35.1-7.18.1.aarch64.rpm ; \
```

SLE_BCI now ships binutils-gold for aarch64, let's use instead
```suggestion
        zypper -n install -y binutils-gold ; \
```